docs/2.guide/3.going-further/9.debugging.md

```diff
@@ -29,37 +29,41 @@ You may need to update the config below with a path to your web browser. For mor
 {
   // Use IntelliSense to learn about possible attributes.
   // Hover to view descriptions of existing attributes.
-  "version": "0.2.0",
-  "configurations": [
+  version: "0.2.0",
```

Comment:
don't we need to keep this JSON quoted?

Reply:
Sorry about that. My prettier extension made some changes I didn't realize.
Fixed in the latest commit.